autorest/src/main/java/com/microsoft/embeddedsocial/autorest/EmbeddedSocialBatchedClientImpl.java

```diff
@@ -41,15 +41,19 @@
     private final ArrayList<Response> batchResps;
     private final int batchSize;
     private final CountDownLatch latch;
-    private final Object syncObject = new Object();
-    private Boolean issueBatchAlreadyCalled = false;
+    private final Object syncObject;
+    private Boolean issueBatchAlreadyCalled;
```

Comment:
s/Boolean/boolean/

Reply:
Also, init the two booleans right away in one shot. See:
https://stackoverflow.com/questions/3918578/should-i-initialize-variable-within-constructor-or-outside-constructor

Reply:
Init the sync object also in one shot.

Reply:
issueBatchAlreadyCalled must be an Object since we synchronize on it in issueBatch()